### ISSUE TITLE
dcm2jpg and wado jpeg problem - white background

### ISSUE THREAD

**Issue by thiagobusa** [dcm4che/dcm4che#504]
When i export the study, some imagens has a white background. In DICOM format, she stay as a normal background (black).

When exported using dcm2jpg tool...
![ExportedJPEG-WhiteBackground](https://user-images.githubusercontent.com/5495168/61636035-327c5d00-ac6b-11e9-8b8f-ea3b53b64108.jpeg)

The same happens when call WADO uri for image/jpeg type...
![Wado-white-background](https://user-images.githubusercontent.com/5495168/61636062-41630f80-ac6b-11e9-844d-dc0fad9d2e3b.png)

However, in the Weasis JPEG exporter, its works...
![Weasis-JPEG-Exported](https://user-images.githubusercontent.com/5495168/61636095-53dd4900-ac6b-11e9-8707-f69cbfbb1c7c.jpeg)

DCM4CHEE ARC and TOOLS version 5.17.1

Weasis version 3.5.2

Images https://drive.google.com/file/d/15Q4kU4G-3g2hTycP6-gdPY4phhLJ4J9A/view?usp=sharing

**Comment by gunterze** [comment]
The artifact is caused by the fact, that the decompressed pixel values have values > 1023, which conflicts with the values of
```
(0028,0101) US #2 [10] BitsStored
(0028,0102) US #2 [9] HighBit
```
`dcm2jpg` and wado jpeg apply a bitmask according the value of Bits Stored, which results in the white background. You get the same result with [DCMTK's dcmj2pnm](https://www.google.com/url?q=https%3A%2F%2Fsupport.dcmtk.org%2Fdocs%2Fdcmj2pnm.html&sa=D&sntz=1&usg=AFQjCNGt8mj_XWcDHgC26vBGtgeF62lUxw).

According [DICOM PS3.5: Valid Values of Pixel Data Related Attributes for JPEG Lossy Transfer Syntaxes using Standard Photometric Interpretations](http://dicom.nema.org/medical/dicom/current/output/html/part05.html#table_8.2.1-1), Bits Stored must be either 8 or 12 - and can't be 10 - for JPEG Extended Transfer Syntax,

**Comment by gunterze** [comment]
Fixed by #512
